apps/api/src/app/integrations/dtos/credentials.dto.ts

```diff
@@ -41,4 +41,6 @@ export class CredentialsDto {
   ignoreTls?: boolean;
   @ApiPropertyOptional()
   tlsOptions?: Record<string, unknown>;
+  @ApiPropertyOptional()
+  baseUrl?: string;
```

Comment:
I find `baseUrl` could be a bit confusing. And as it is a specific credential property added for this feature what do you think of naming it `pushWebhookUrl`? I was going to suggest `webhookUrl` in case we wanted in the future something similar with the other channels, but not sure if that's a direction we will want to take.
@scopsy your thoughts about this?

Reply:
Yeah i used the same as the other webhook proposal. I can change it to whatever you prefer :)

Reply:
@p-fernandez iirc, I think it was listed as a credential property somewhere but not totally implemented. (this is why i used this property naming but could rename it to whatever you want btw)

Reply:
I changed to the already existing webhookUrl